lib/ActionSheetCustom.js

```diff
@@ -68,14 +68,16 @@ class ActionSheet extends React.Component {
     Animated.timing(this.state.sheetAnim, {
       toValue: 0,
       duration: 250,
-      easing: Easing.out(Easing.ease)
+      easing: Easing.out(Easing.ease),
+      useNativeDriver: this.props.useNativeDriver || true
```

Comment:
suggest change to `this.props.useNativeDriver === true`

Reply:
Thanks. This will need `useNativeDriver` to be explicitly passed to the component. So instead I added a `defaultProp`.